src/List.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class List extends React.PureComponent {
+  state = {
+    todo: this.props.todo,
+    users: this.props.user,
+    checkboxStatus: false,
```

Comment:
name like `checkboxStatus` assumes that it holds some value related to status (like `completed`, `pending` or something else), if you want to store a boolean value, you need to name the variable accordingly (like `isTodoCompleted`, like a question with yes-no answer)

Reply:
fixed